internal/uplink/heartbeat.go

```diff
@@ -0,0 +1,11 @@
+// package uplink defines the types used in groundstation <-> satiate transport.
+
+package uplink
+
+import "time"
+
+const HeartbeatUrl = "/gs-api/heartbeat/"
+
+type HeartbeatReq struct {
+	Time time.Time
```

Comment:
I think @danielg0 might have mentioned this already, but using the time that a request hits the actual server's endpoint is likely ideal here. Otherwise, time has no ground truth.

Reply:
punted to #32, because this is more about comunication than accurate timekeeping